content/applications/marketing/events/create_events.rst

```diff
@@ -0,0 +1,427 @@
+   The :guilabel:`Social Post` option **only** appears if the *Social Marketing* application is
+   installed. The :guilabel:`WhatsApp` option **only** appears if the *WhatsApp Integration* module
+   is installed.
+
+   :doc:`WhatsApp <../../productivity/whatsapp>` templates **cannot** be edited during active
+   configuration. A separate approval from *Meta* is requried.
+
+Then, either select a communication template from the :guilabel:`Template` drop-down menu.
+
+.. note::
+   These existing communication templates can be modified directly from this field, if necessary, by
+   selecting the chosen template, and clicking the :guilabel:`➡️ (right arrow)` icon to the right.
+   Doing so reveals a separate page where users can edit the :guilabel:`Content`, :guilabel:`Email
+   Configuration`, and :guilabel:`Settings` of that particular communication template.
+
+.. tip::
+   To create a new communication template from the :guilabel:`Template` field in the
+   :guilabel:`Communication` tab, start typing in the name of the new template, then click
+   :guilabel:`Create and edit...` from the resulting drop-down menu. Doing so reveals a blank
+   :guilabel:`Create Template` pop-up window, wherein users can create a custom communication
+   template.
+
+   To learn more, refer to the :ref:`Create an email <email_marketing/create_email>` documentation.
+
+Next, determine the :guilabel:`Interval`, :guilabel:`Unit`, and :guilabel:`Trigger` from their
+respective drop-down fields, letting Odoo know when the communication should be sent.
+
+The user enters a desired number into the :guilabel:`Interval` field. Then, select a
+:guilabel:`Unit` from the drop-down menu. The options are: :guilabel:`Immediately`,
+:guilabel:`Hours`, :guilabel:`Days`, :guilabel:`Weeks`, and :guilabel:`Months`.
+
+Then, select an option from the :guilabel:`Trigger` drop-down menu. The options are:
+:guilabel:`After each registration`, :guilabel:`Before the event`, and :guilabel:`After the event`.
+
+The figures in the :guilabel:`Sent` column will populate as communications are sent. And, beside the
+number present in the :guilabel:`Sent` column there are different icons that appear, depending on
+the status of that particular communication.
+
+The status of *Running* is represented by a :icon:`fa-cogs` :guilabel:`(three gears)` icon. The
+status of *Sent* is represented by a :icon:`fa-check` :guilabel:`(checkmark)` icon. And, the status
+of *Scheduled* is represented by an :icon:`fa-hourglass-half` :guilabel:`(hourglass)` icon.
+
+To delete any communication from the :guilabel:`Communication` tab, simply click the
+:icon:`fa-trash-o` :guilabel:`(trash can)` icon on the corresponding communication line. Doing so
+removes the communication from the event entirely, and furthermore, stops that specific
+communication from running, if it was active or pending.
+
+There is no limit to how many communications can be added in the :guilabel:`Communication` tab of an
+event form.
```

Comment:
Remember my suggestion to remove these statements to the top of the section, and the expand on the Mail communication by providing an example, as well as a note for editing templates in this view or managing all templates.

IMO by placing these at the bottom of the section, the reader can skim through the initial set up, and land on actionable examples and further reading
```suggestion
.. example::
   To send a confirmation email an hour after an attendee registers for an event, configure the
   following communication:

   - :guilabel:`Interval`: `1`
   - :guilabel:`Unit`: :guilabel:`Hours`
   - :guilabel:`Trigger`: :guilabel:`After each registration`

.. note::
   Existing email templates can be modified directly from the :guilabel:`Template` drop-down menu,
   if necessary, by clicking the :icon:`oi-arrow-right` :guilabel:`(right arrow)` icon next to the
   template name. Doing so reveals a separate page where users can edit the :guilabel:`Content`,
   :guilabel:`Email Configuration`, and :guilabel:`Settings` of that particular email template.

   To view and manage all email templates, activate :ref:`developer-mode` and navigate to
   :menuselection:`Settings --> Technical --> Email: Email Templates`. Modify with caution as email
   templates effect all communications where the template is used.
```